app/components/channel_list/header/header.tsx

```diff
@@ -57,19 +58,54 @@ const getStyles = makeStyleSheetFromTheme((theme: Theme) => ({
 
 const ChannelListHeader = ({displayName, iconPad}: Props) => {
     const theme = useTheme();
+    const isTablet = useIsTablet();
     const serverDisplayName = useServerDisplayName();
     const marginLeft = useSharedValue(iconPad ? 44 : 0);
     const styles = getStyles(theme);
     const animatedStyle = useAnimatedStyle(() => ({
         marginLeft: withTiming(marginLeft.value, {duration: 350}),
     }), []);
 
-    const intl = useIntl();
-
     useEffect(() => {
         marginLeft.value = iconPad ? 44 : 0;
     }, [iconPad]);
 
+    const onPress = useCallback(() => {
+        const renderContent = () => {
+            return (
+                <>
+                    <PlusMenuList
+                        pickerAction='browseChannels'
+                        onPress={() => {
+                            //this is a click
+                        }}
+                    />
+                    <PlusMenuList
+                        pickerAction='createNewChannel'
+                        onPress={() => {
+                            //this is a click
+                        }}
+                    />
+                    <PlusMenuList
+                        pickerAction='openDirectMessage'
+                        onPress={() => {
+                            //this is a click
+                        }}
+                    />
+                </>
+            );
+        };
+
+        const closeButtonId = 'close-plus-menu';
+        bottomSheet({
+            closeButtonId,
+            renderContent,
+            snapPoints: [4 * ITEM_HEIGHT, 10],
+            theme,
+            title: 'Plus Menu',
```

Comment:
This title shows on Tablets and needs to be localized. This text of this title does not look right to me. @Willyfrog can you work with UX to determine what would be a good title for this.

This is an example on a Tablet for the BottomSheet that opens when you tap on the + icon in the team sidebar
<img width="600" alt="image" src="https://user-images.githubusercontent.com/6757047/153581990-a779bfa6-2388-4a02-be11-0f10f81f359a.png">

Reply:
Ok waiting for this